infrastructure/disk.tf

```diff
@@ -114,9 +114,51 @@ resource "aws_s3_bucket" "data_refinery_compendia_bucket" {
   }
 }
 
-resource "aws_s3_bucket_metric" "compendia_metrics" {
+resource "aws_s3_bucket_metric" "compendia_bucket_metrics" {
   bucket = "${aws_s3_bucket.data_refinery_compendia_bucket.bucket}"
-  name   = "data-refinery-compendia-metric-${var.user}-${var.stage}"
+  name   = "EntireBucket"
```

Comment:
EntireBucket?

Reply:
Not sure what that means and shouldn't this be namespaced with the user/stack so that there won't be conflicts between dev/staging/prod?

Reply:
Yeah... funny that, the bucket parameter is where tell it which bucket to create a metrics rule for (which is dev/stage/prod) depending on env, EntireBucket is some codeword they use that allows us to enable object level logs.

Reply:
Ah, in that case can you leave a comment explaining that?

Reply:
updated the comment